src/scripts.js

```diff
@@ -49,6 +51,44 @@ function onStartup() {
     .catch(err => console.log(err.message)) 
 }
 
+function addRecipe(event) {
+  let recipeToAdd = data.recipeData.find(recipe =>recipe.id === Number(event.target.id));
+  user.addToMealList(recipeToAdd);
+  console.log(user.mealList);
+}
+
+function displayAddedMeals() {
+  if (cardArea.classList.contains('all')) {
+    cardArea.classList.remove('all')
+  }
+  if (!user.mealList.length) {
+    mealButton.innerHTML = 'You have no meals yet';
+    populateCards(cookbook.recipes);
+    return
+  } else {
+    mealButton.innerHTML = 'Refresh Meals'
+    cardArea.innerHTML = '';
+    user.mealList.forEach(recipe => {
```

Comment:
we might be able to use the populateCards function with our mealList as an argument here